README.md

```diff
@@ -2,102 +2,102 @@
 
 ## Contents
 
-- [Client Drivers (official)](#client-drivers-official)
-- [Client Drivers (community)](#client-drivers-community)
-- [IDE and editors support](#ide-and-editors-support)
 - [Open source projects using TypeDB](#open-source-projects-using-typedb)
 - [Conferences](#conferences)
 - [Documentation](#documentation)
 - [Example projects](#example-projects)
 - [Blog posts](#blog-posts)
 - [Videos](#videos)
+- [Client Drivers (official)](#client-drivers-official)
+- [Client Drivers (community)](#client-drivers-community)
+- [IDE and editors support](#ide-and-editors-support)
 - [Community](#community)
 
-
-## Client Drivers (official)
-
-List of officially supported drivers to connect with Grakn Server.
-
-- [`client-java`](https://github.com/vaticle/typedb-client-java)
-- [`client-nodejs`](https://github.com/vaticle/typedb-client-nodejs)
-- [`client-python`](https://github.com/vaticle/typedb-client-python)
-
-
-## Client Drivers (community)
-
-- [`client-elixir`](https://github.com/taxfix/grakn_elixir)
-- [`Grakn.Net`](https://github.com/WalternativE/Grakn.Net)
-- [`TypeDBClient.jl`](https://github.com/Humans-of-Julia/TypeDBClient.jl)
-
-
-## IDE and editors support
-
-- [`typeql-plugin-emacs`](https://github.com/typedb-osi/typeql-plugin-emacs) - Major mode that provides syntax highlighting and basic completion for TypeQL.
-- [`typeql-plugin-vscode`](https://github.com/typedb-osi/typeql-plugin-vscode) - Visual Studio Code support for TypeQL.
-- [`typeql-plugin-jetbrains`](https://github.com/typedb-osi/typeql-plugin-jetbrains) - JetBrains support for TypeQL.
-- [`typeql-plugin-atom`](https://github.com/typedb-osi/typeql-plugin-atom) - Atom's syntax highlighting for TypeQL.
-
-
 ## Open source projects using TypeDB
 
-- [`TypeDB OSI`](https://github.com/typedb-osi) - TypeDB OSI Ecosystem comprises all resources and technologies to be used with TypeDB.
-- [`TypeDB Loader`](https://github.com/bayer-science-for-a-better-life/grami) - Formerly "GraMi", an open-source data migrator for loading data into TypeDB at scale.
+- [`KGLIB - Machine Learning with TypeDB`](https://github.com/vaticle/kglib) - KGLIB provides tools to enable machine learning with TypeDB.
+- [`TypeDB Studio - IDE for TypeDB`](https://github.com/vaticle/typedb-studio) - TypeDB Studio is an Integrated Development Environment for TypeDB for performing knowledge engineering.
+- [`TypeDB OSI - Cyber Threat Intelligence`](https://github.com/typedb-osi/typedb-cti) - TypeDB CTI is an open source threat intelligence platform enabling threat intel professionals to bring together their disparate CTI information into one database 
+and find new insights about cyber threats.
+- [`TypeDB OSI - Biomedical Knowledge Graph`](https://github.com/typedb-osi/typedb-bio) - TypeDB Bio is an open source biomedical knowledge graph providing biomedical researchers an intuitive way to query interconnected and heterogeneous biomedical 
```

Comment:
We bring these examples forward - someone particularly interested in cyber threat intelligence or the biomedical field won't currently have anything to immediately draw their eye unless they explore every link.